index.html

```diff
@@ -132,7 +132,8 @@ <h2>Seguinos por Instagram</h2>
                     <p><strong>Follow</strong> y sumate a nuestros seguidores para no perderte<br>nuestras recetas semanales, y participar de increíbles premios!</p>
                     <h4><strong>Click</strong> en el logo para seguirnos!</h4>
                     <a href="https://www.instagram.com/holapanzasana/">
-                        <img class="logoinsta hvr-pulse-grow" src="assets/images/instagram.svg"></a>
+                        <img class="logoinsta hvr-pulse-grow" src="assets/images/instagram.svg">
```

Comment:
same here.. plz check all the tags that do not have a close tag and perform something similar to what I showed u above!

Reply:
Also done